python/mlcroissant/mlcroissant/_src/operation_graph/operations/field.py

```diff
@@ -31,6 +31,9 @@ def _cast_value(value: Any, data_type: type | term.URIRef | None):
             return deps.PIL_Image.open(io.BytesIO(value))
         else:
             raise ValueError(f"Type {type(value)} is not accepted for an image.")
+    elif data_type == DataType.AUDIO_OBJECT:
+        output = deps.LIB_Audio.load(io.BytesIO(value))
+        return str([output[0].tolist(), output[1]])
```

Comment:
Why a str? How will you use this in an ML pipeline? What would be the most useful signal to return here?

Reply:
I did it like that because the output needs to have double quotes. I'm about to push a version that just outputs the regular librosa output, but I don't know if it will pass the test.